WLogger/WeekLogs/WeekLogsTableViewController.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  WeekLogsTableViewController.swift
+//  WLogger
+//
+//  Created by Pablo Giorgi on 1/10/16.
+//  Copyright © 2016 Wolox. All rights reserved.
+//
+
+import UIKit
+
+class WeekLogsTableViewController: UITableViewController {
+
+    var viewModel: WeekLogsTableViewModel?
+}
+
+extension WeekLogsTableViewController {
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        self.refreshControl = UIRefreshControl()
+        self.tableView.addSubview(refreshControl!)
+        self.refreshControl?.addTarget(self, action: "handleRefresh:", forControlEvents: UIControlEvents.ValueChanged)
+        tableView.registerNib(UINib(nibName: "LogTableViewCell", bundle: nil), forCellReuseIdentifier: LogTableViewCellIdentifier)
+        
+        viewModel?.fetchLogs.executing.producer.startWithNext { executing in
+            if executing {
+                // Show spinner
+            } else {
+                // Hide spinner
+            }
+        }
+        
+        viewModel?.fetchLogs.apply("").start { [unowned self] event in
+            switch event {
+            case .Next(_):
+                print("Before reload table....")
```

Comment:
Yeah that could happen because mutation order is not guarantedd. Change all this to

```swift
viewModel?.logs.producer.startWithNext { _ in self.tableView.reloadData() }
viewModel?.fetchLogs.apply("").startWithFailed { error in
  // TODO handle error
}
```